packages/app/src/DashboardPage.tsx

```diff
@@ -320,7 +320,7 @@ const Tile = forwardRef(
           </div>
         ) : (
           <div
-            className="fs-7 text-muted flex-grow-1 overflow-hidden"
+            className="fs-7 text-muted flex-grow-1"
```

Comment:
The only thing I can think of that we added this for is possibly for Y axis overflow (like a million legend items overflowing the tile). Though at this point I think we have legend overflow handling as well so should be safe to remove, unless we still need a overflow y hidden for some reason.